include/librealsense2/dds/dds-device.cpp

```diff
@@ -167,6 +167,61 @@ dds_device::foreach_motion_profile( size_t sensor_index,
     return msg->second.num_of_profiles;
 }
 
+void dds_device::sensor_open( size_t sensor_index, const std::vector< rs2_video_stream > & profiles )
```

Comment:
Shouldn't there be only one profile? Why are we passing multiple?

Reply:
sensor_interface gives you the option of opening multiple streams, I wanted to support this ability.

I guess it's because one sensor can have multiple streams that can be opened, like infrared+depth